### PR TITLE
Checkstyle, findbugs, lint, NonNull.

### PR DESCRIPTION
**Pull request by kejmil01**

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
     <application
         android:allowBackup="true"
+        android:fullBackupContent="@xml/nearby_places_backup_scheme"
```
Comment (michal1990) [inline app/src/main/AndroidManifest.xml]: hmm I believe that our app is not called nearby places :P

File: app/build.gradle [hunk 2]
```diff
@@ -17,13 +18,23 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
+    lintOptions {
+        disable 'InvalidPackage'
```
Comment (BartSoftware) [inline app/build.gradle]: redundant

File: app/build.gradle [hunk 3]
```diff
@@ -17,13 +18,23 @@ android {
...
+        disable 'GoogleAppIndexingWarning', 'UnusedResources', 'MissingConstraints'
```
Comment (BartSoftware) [inline app/build.gradle]: reduntant

File: app/lint.xml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<lint>
+    <issue id="RtlCompat" severity="ignore" />
```
Comment (BartSoftware) [inline app/lint.xml]: redundant

File: config/findbugs/findbugs-excluded.xml [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+  </Match>
+  <Match>
+    <Or>
+      <Class name="~.*\.gradle\$.*"/>
+      <Class name="~.*\.R\$.*" />
+      <Class name="~.*\.Manifest\$.*" />
+    </Or>
+  </Match>
+  <Match>
+      <Class name="~.*\$*Kt.*" /> <!-- exclude Kotlin auto-generated classes -->
```
Comment (michal1990) [inline config/findbugs/findbugs-excluded.xml]: it's redundant - we don't have kotlin files

**Review by BartSoftware — APPROVED** [review]

**Pull request merged**